app/views/partials/multiple_repayment_details.scala.html

```diff
@@ -59,7 +59,7 @@
             } else{
             <span aria-hidden="true">@{LangMessages.`View progress`.show}</span>
             }
-            <span class="visually-hidden">@{LangMessages.`Accounting period`.show} @onePayment.period</span>
+            <span class="visually-hidden">@{LangMessages.`Accounting period hidden`.show} @onePayment.period</span>
```

Comment:
Why would the `View progress` link have assistive text that says "View history for [...]"?